wgpu-core/src/command/mod.rs

```diff
@@ -596,6 +596,8 @@ pub enum CommandEncoderError {
     InvalidTimestampWritesQuerySetId(id::QuerySetId),
     #[error("Attachment TextureViewId {0:?} is invalid")]
     InvalidAttachmentId(id::TextureViewId),
+    #[error(transparent)]
+    InvalidColorAttachment(#[from] ColorAttachmentError),
```

Comment:
Should we inline `TooMany` as it's the only variant that is used?

Reply:
I think it's fine as it is, it's not the only place where we only use a subset.